### PR TITLE
Updates the format of documentation to match neeto-commons-frontend style.

### LINKED ISSUE #32: Update the format of documentation to match neeto-commons-frontend style
https://github.com/bigbinary/neeto-commons-frontend/pull/898 introduces a new format for documentation where the documentation for each function is written in a separate file. 

---

**Action Item**

- [ ] Migrate the changes in https://github.com/bigbinary/neeto-commons-frontend/pull/898.

documentation _t low-priority _t

### PR DESCRIPTION
**Pull request by deepakjosp**
- Fixes #32

#### Checklist

- [x] I have made corresponding changes to the documentation
- [x] I have verified the generated JSDoc comments
- [ ] ~I have updated the types definition of modified exports~
- [ ] ~I have added jest tests for newly added changes~
- [x] I have verified the functionality in some of the neeto web-apps.

### CODE REVIEW

**Comment by deepakjosp** [comment]
@ajmaln  _a please review.

**Comment by ajmaln** [comment]
@josephmathew900 _a Please review

**Comment by josephmathew900** [comment]
@AbhayVAshokan _a please review.

File: docs/general/building-and-releasing.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Building and releasing.
```
Comment (AbhayVAshokan) [inline docs/general/building-and-releasing.md]:
```suggestion
# Building and releasing
```

File: docs/general/api-documentation-logistics.md
Comment (AbhayVAshokan) [inline docs/general/api-documentation-logistics.md:1]: Please link this file to the end of README.md

**Review by AbhayVAshokan — APPROVED** [review]
@deepakjosp _a: https://abhay-ashokan.neetorecord.com/watch/c90bae8b-e689-40b6-bc05-03f032f2aa80

LGTM! Please merge it yourself after this change.

**Pull request merged**